msteams-platform/samples/approvals-app-api.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Approvals app APIs
+description: Learn the different types of the approvals app Microsoft Graph APIs for personal and third-party apps, its use cases, and benefits.
+ms.topic: reference
+ms.localizationpriority: medium
+ms.author: v-sdhakshina
+ms.date: 07/10/2024
+---
+
+# Approvals app APIs
+
+> [!NOTE]
+> Approvals app API is available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
+
+The approvals app API in Microsoft Teams allows you to integrate approval workflows into both Teams app and third-party apps. You can create new approvals, update existing ones, and list all approvals. It mentions the ability to create, update, and list approvals, assign approvals to users, specify custom responses, and receive notifications on Teams. As APIs support graph subscriptions, you can send notifications for approval responses. Approvals app is available as a personal app for all Teams users.
+
+The following table provides the list of APIs available across Microsoft Graph for the approvals app:
+
+| API | Description |
+| ---- | ---- |
+| [Approval item](/graph/api/resources/approvalitem?view=graph-rest-beta&preserve-view=true) | Represents a request for approval, encompassing related decisions, status, and responses. You can create new approval items, view sent or received approval items, and perform actions on these approval items. |
+| [Approval solution](/graph/api/resources/approvalsolution?view=graph-rest-beta&preserve-view=true) | Represents the provisioning status of the approval solution for a tenant. |
+| [Approval item request](/graph/api/resources/approvalitemrequest?view=graph-rest-beta&preserve-view=true) | Represents a request created for each approver on `approvalItem`. |
+| [Approval item response](/graph/api/resources/approvalitemresponse?view=graph-rest-beta&preserve-view=true) | Represents a response to an approval item request. |
+| [Approval item view point](/graph/api/resources/approvalitemviewpoint?view=graph-rest-beta&preserve-view=true) |Represents the user's roles for an `approvalItem`.|
+| [Approval operation](/graph/api/resources/approvaloperation?view=graph-rest-beta&preserve-view=true) | Represents the status and details for an operation performed on `approvalItem`.|
```

Comment:
Do we have sample code for this? If not, please share it with sample code team in channel.